lib/src/widgets/utils/ds_card.widget.dart

```diff
@@ -78,6 +82,9 @@ class DSCard extends StatelessWidget {
       case DSMessageContentType.contact:
         return _buildContact();
 
+      case DSMessageContentType.reply:
+        return _resolveWidget(messageType: content['replied']['type']);
```

Comment:
A simpler solution would be to return ```DSCard``` here passing ```type: content['replied']['type']``` and ```content: content['replied']['value']``` as parameters. Then, you could undo the other changes, since it will follow the same path as any other message.